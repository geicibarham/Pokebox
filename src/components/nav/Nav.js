
import styles from './nav.module.css'
const Nav = () => {
    return (
        <section className={styles.nav}>
            <h1>Welcome To Pokebox!</h1>
        </section>
    )
}

export default Nav;