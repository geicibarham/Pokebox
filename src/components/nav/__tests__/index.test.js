import '@testing-library/jest-dom/extend-expect';
import { render, cleanup } from '@testing-library/react';
import Nav from '../Nav';
import React from 'react';
afterEach(cleanup);

describe('Nav component', () => {
    it('renders', () => {
        render(<Nav />);
    });

    it('matches snapshot DOM node structure', () => {
        const { asFragment } = render(<Nav />);

        expect(asFragment()).toMatchSnapshot();
    });
})
