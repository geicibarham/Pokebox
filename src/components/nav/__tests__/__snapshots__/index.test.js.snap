// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <nav
    class="nav"
  >
    <h1>
      Welcome To Pokebox!
    </h1>
  </nav>
</DocumentFragment>
`;
